presto-docs/src/main/sphinx/cache/library.rst

```diff
@@ -0,0 +1,66 @@
+=====================
```

Comment:
I would have liked to start by saying Presto supports local data cache and then introduce Alluxio. Not a strong opinion.

Reply:
added